src/styles/components/_home-search-suggestion.scss

```
@@ -0,0 +1,128 @@
+.home-search-suggestion {
+  &__autocomplete,
+  &__not-found {
+    all: unset;
+    position: absolute;
```

From other `scss` classes, we normally put empty space after `all: unset` 
```suggestion
    all: unset;
    
    position: absolute;
```